### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by irsath-nguyen-ultra**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at irsath.nguyen@ultra.io. All
```
Comment (romain-gilles-ultra) [inline CODE_OF_CONDUCT.md:58]: I think you should not put your email here ;)

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
```
Comment (romain-gilles-ultra) [inline CODE_OF_CONDUCT.md:64]: I do understand it for an open source project but not sure in our case

Comment (romain-gilles-ultra) [reply CODE_OF_CONDUCT.md:64]: Maybe if we want to open-source the project

Comment (irsath-nguyen-ultra) [reply CODE_OF_CONDUCT.md]: You're right, I made some changes

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (irsath-nguyen-ultra) [reply CODE_OF_CONDUCT.md]: Yes

**Pull request merged**